examples/bloom/__main__.py

```diff
@@ -16,6 +18,10 @@
     DataOwner("genebank-6", NUM_FEATURES, TRAINING_SET_SIZE, TEST_SET_SIZE),
 ]
 
+logging.basicConfig()
+logger = logging.getLogger(__name__)
```

Comment:
Using `__name__` here instead of `tf_encrypted` might prevent `logger.setLevel(logging.DEBUG)` below from having an effect. Same for the other files.

Alternatively it might be okay to simply do `logging.basicConfig(level=logging.DEBUG)` here, if it doesn't produce too much debugging information from other packages.

Reply:
You're right, I usually set the level in the call to basicConfig. Probably best to do that. And if you have too much output, you can still increase the logging level from the other loggers. 
So which solution do you prefer?

Reply:
would you mind giving `logging.basicConfig(level=logging.DEBUG)` a try and see if it prints a lot when run? if not then let's go with that.

Reply:
gave it a try and it looks good; nothing extra printed